<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>LS</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Unit</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="vallineEdit_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="vallineEdit_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="vallineEdit_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="vallineEdit_4"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>29</y>
         <width>51</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="unitlineEdit_1">
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="unitlineEdit_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="unitlineEdit_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="unitlineEdit_4"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>181</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Param</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Init Val</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="initFlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>StartX</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startxspinBox">
          <property name="maximum">
           <number>10000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>StartY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startyspinBox">
          <property name="maximum">
           <number>10000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>InitAngle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="iniangledoubleSpinBox">
          <property name="minimum">
           <double>-99999999.989999994635582</double>
          </property>
          <property name="maximum">
           <double>99999999.989999994635582</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="angledoubleSpinBox">
          <property name="minimum">
           <double>-9999999.990000000223517</double>
          </property>
          <property name="maximum">
           <double>99999999.989999994635582</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Iter times</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="iterspinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Draw Len</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lenspinBox"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="startpushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>590</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>IFS</string>
     </attribute>
     <widget class="QPushButton" name="drawIFSpushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>IFS_Draw</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>135</width>
        <height>172</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>StartX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startxIFSlineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>StartY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startyIFSlineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>CX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cxIFSlineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>CY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cyIFSlineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_Parama"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_Parama">
   <property name="text">
    <string>Save Param</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
